CapacityManagement/DashboardGenerator/Create-AzSStorageDashboard.ps1

```diff
@@ -0,0 +1,544 @@
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('all', 'object', 'infrastructure', 'vmtemp')]
+    [string]$volumeType = 'all'
+)
+
+<#
+.SYNOPSIS
+    Generate json used in Azure Stack portal to show volumes performances.
+.Description
+    This function is used to generate dashboard jsons, which can be uploaded on Azure Stack Dashboard to create related dashboards.
+    You can use this function without parameters to create jsons, showing last 1 day metrics, at the current folder.
+    To set time range, you can define duration or pair of startTime and endTime.
+.Inputs
+    Charts' Time range and granularity settings. Json output location setting.
+.Outputs
+    Three jsons represent count, latency and throughput performance respectively.
+.Parameter timeGrain
+    The timespan defines the time granularity in charts, in ISO 8601 duration format.
+.Parameter outputLocation
+    The location to expose generated jsons.
+.Parameter duration
+    The timespan defines the time range of volume metrics, in ISO 8601 duration format.
+.Parameter startTime
+    The start time of time range shown on dashboard.
+.Parameter endTime
+    The start time of time range shown on dashboard.
+.Example
+    # default json save to spedified location
+    Save-AzureStackVolumesPerformanceDashboardJson -outputLocation 'D:\dashboardJsons'
+.Example
+    # data of last day with 15min interval
+    Save-AzureStackVolumesPerformanceDashboardJson -duration "P1D" -timeGrain "PT15M"
+.Example
+    # date from 4/1 to 4/8 with 1hr interval 
+    Save-AzureStackVolumesPerformanceDashboardJson -startTime (Get-date("2019-04-01")) -endTime (Get-date("2019-04-08")) -timeGrain "PT1H"
+.Notes
+    Author: Azure Stack Azure Monitor Team
+.Link
+    Source Code: https://github.com/GTMer/AzureStack-VolumesPerformanceDashboard-Generator
+#>
+function Save-AzureStackVolumesPerformanceDashboardJson {
+    [CmdletBinding(DefaultParameterSetName="relativeTime")]
+    param (
+        [Parameter(Mandatory = $false)]
+        [System.Object]$DefaultProfile,
+        [Parameter(ParameterSetName="relativeTime")]
+        [ValidateSet('PT30M', 'PT4H', 'PT12H', 'P1D', 'P2D', 'P3D', 'P7D', 'P30D')]
+        [string]$duration = 'P1D',
+        [Parameter(Mandatory=$True, ParameterSetName="absoluteTime", HelpMessage="Please enter the start time of time range you want to see.")]
+        [datetime]$startTime,
+        [Parameter(Mandatory=$True, ParameterSetName="absoluteTime", HelpMessage="Please enter the end time of time range you want to see.")]
+        [datetime]$endTime,
+        [Parameter(Mandatory = $false)]
+        [ValidateSet('Automatic', 'PT1M', 'PT1H', 'P1D', 'PT5M', 'PT15M', 'PT30M', 'PT6H', 'PT12H')]
+        [string]$timeGrain = 'Automatic',
+        [Parameter(Mandatory = $false)]
+        [string]$outputLocation = '.'
+    )
+
+    # check if AzureRM Module is present
+    $AzureRMModule = Get-command -Name Get-AzureRmContext -ErrorAction SilentlyContinue
```

Comment:
this should change to Get-AzContext , there should not be any reference to -AzureRM* in this script and -AzureRM should be replaced with -Az